test/jdk/javax/net/ssl/SSLEngine/Basics.java

```diff
@@ -77,34 +95,44 @@ public static void main(String args[]) throws Exception {
         System.out.println(ssle);
 
         String [] suites = ssle.getSupportedCipherSuites();
-        String secondSuite = suites[1];
-        String [] oneSuites = new String [] { secondSuite };
+        // sanity check that the ciphersuite we want to use is still supported
+        Arrays.stream(suites)
+                .filter(s -> s.equals(cipherSuite))
+                .findFirst()
+                .orElseThrow((() ->
+                        new RuntimeException(cipherSuite +
```

Comment:
Test throws the Exception, not RuntimeException in other failure cases in this test. Can you please update those to RuntimeException as well?